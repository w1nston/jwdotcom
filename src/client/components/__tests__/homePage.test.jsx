import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import HomePage from '../homePage';

describe('<HomePage/>', () => {
  it('renders', () => {
    const component = shallow(<HomePage />);
    expect(toJson(component)).toMatchSnapshot();
  });
});
