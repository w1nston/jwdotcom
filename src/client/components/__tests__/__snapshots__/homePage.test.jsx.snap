exports[`<HomePage/> renders 1`] = `
<SideEffect(DocumentTitle)
  title="Home">
  <section
    className="main-content">
    <h1>
      Home...
    </h1>
    <p>
      Figure something out what to place here...
    </p>
  </section>
</SideEffect(DocumentTitle)>
`;
